automation/gradle/versionCode.gradle

```diff
@@ -0,0 +1,45 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import java.text.SimpleDateFormat
+
+// This gradle scripts generates a "unique" version code for our release versions.
+//
+// The result of the version code depends on the timezone. We assume that this script will only be used
+// for release versions and running on our build servers with a fixed timezone.
+//
+// The version code is composed like: yDDDHHmm
+//  * y   = Double digit year, with 17 substracted: 2018 -> 18 -> 1
+//  * DDD = Day of the year, pad with zeros if needed: September 6th -> 249
+//  * HH  = Hour in day (00-23)
+//  * mm  = Minute in hour
+//
+// For September 6th, 2018, 9:41 am this will generate the versionCode: 12490941 (1-249-09-41).
+//
+// Note that we only use this generated version code for builds we want to distribute. For local
+// debug builds we use a fixed versionCode to not mess with the caching mechanism of the build
+// system.
+
+ext {
+    def base = "1"
+    def today = new Date()
+
+    // We use the current year (double digit) and subtract 17. We first released Reference Browser in
+    // 2018 so this value will start counting at 1 and increment by one every year.
+    def year = String.valueOf((new SimpleDateFormat("yy").format(today) as int) - 17)
```

Comment:
I think for a different setup I have used the exact year here to use "0" and have one less digit (well, until next month, but still -1). Can you try that?